samples/hosting-autoscaling-on-sagemaker/hosting-autoscaling-custom-metric.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: applicationautoscaling.services.k8s.aws/v1alpha1
+kind: ScalableTarget
+metadata:
+  name: m-target-may13-suspended
```

Comment:
make `m-*` strings generic and add placeholders for values which need to be replaced